pthread_support.c

```diff
@@ -394,6 +394,7 @@ STATIC pthread_t GC_mark_threads[MAX_MARKERS];
 {
     int i;
     pthread_attr_t attr;
```

Comment:
I don't know how signals works in MS-Windows.
But if GC marker thread also steals/drops user signals in `win32_thread`, I think similar patch might be required for `win32_thread`.